doc/README.rst

```diff
@@ -16,6 +16,10 @@ and to view it, do::
 
     firefox _build/html/index.html
 
+If you get **mpmath** error, install python3-mpmath package::
+
+    apt-get install python3-mpmath
+
```

Comment:
Please move these to a different PR.

Reply:
Sure will do, don't know how to move them out but I guess I can make another commit undoing these changes, and then redo them from another branch.